constant/constants.go

```diff
@@ -157,6 +158,130 @@ const (
 	PetRemoveExpire byte = 2
 )
 
+const (
+	MiniRoomCreate        byte = 0
```

Comment:
Thanks for pulling these constants but I can't see the room.go update to show these being moved and renamed?